DOCS-1340: Update Split Payments documentation (#2057)
MultiSafepay/docs: content/payments/features/split-payments/_index.md

    - /tools/split-payments/how-do-i-refund-split-payment-orders
    - /payments/features/split-payments/
---

> Split Payments lets you divide an incoming transaction amount between two or more [partner/primary](/account/account-types/) or [affiliate](/account/account-types/) accounts, e.g. as a fee for using your platform.

Reviewer: ```suggestion
Split Payments lets you divide an incoming transaction amount between two or more MultiSafepay accounts. This feature is supported for all [account types](/account/account-types/) and affiliations, e.g.:

- Partner/primary accounts, e.g. for mixed baskets where customers buy products from multiple affiliated merchants in a single transaction on your marketplace
- Affiliated accounts, e.g. for franchises splitting 10% of all incoming transactions to the linked primary account
```

You can split payments by percentage, a fixed amount, or by both.

## Integration

> Split Payments are made by [appending an object specifying the split](/api/#split-payments-orders) to the payload of the API request to create an order.

Reviewer: ```suggestion
To split a payment, [add an object specifying the split](/api/#split-payments-orders) to the payload of a `POST /orders` request.

```

> Split Payments can be created by:
> 
> - Affiliated accounts, e.g. for franchises splitting 10% of all incoming transactions to the linked primary account
> - Partner/primary accounts, e.g. for mixed-baskets where customers buy products from multiple affiliated merchants in a single transaction on a marketplace. 
> 
> Incoming transaction amounts can be split between all account types and affiliations.

Reviewer: ```suggestion
```

Reviewer: Move this up to the introduction.

## Refunds

> You can only refund Split Payments (in full or in part) from the account that created the Split Payment. 

Reviewer: ```suggestion
You can process full and partial refunds, but only from the account that created the split payment. 

```

> To refund a Split Payment, follow these steps:

Reviewer: ```suggestion
To refund a split payment, follow these steps:

```

Reviewer: My take: Split Payments is the name of the feature. Each instance is simply a split payment. What do you think?

Author: Yeah, I like that. In some contexts it's a bit tricky to distinguish between feature or instance of the feature, like in [this line](https://github.com/MultiSafepay/docs/pull/2057/files#diff-a63a92a3e1d5e881c174e66262714ceda807b73bb493550eaaaa760b13865074R35), but I'll try my best ;)
